apps/timetable/ui/js/index.js

```diff
@@ -65,26 +55,19 @@ define(['gh.core', 'gh.subheader', 'gh.calendar', 'gh.student-listview', 'jquery
      * @private
      */
     var setUpCalendar = function() {
+        // Render the calendar template
         gh.api.utilAPI.renderTemplate($('#gh-calendar-template'), {
             'data': null
         }, $('#gh-main'));
 
-        if (gh.data.me.anon) {
-            $(document).trigger('gh.calendar.init');
-        } else {
-            var range = gh.api.utilAPI.getCalendarDateRange();
-            gh.api.userAPI.getUserCalendar(gh.data.me.id, range.start, range.end, function(err, data) {
-                if (err) {
-                    gh.api.utilAPI.notification('Fetching user calendar failed.', 'An error occurred while fetching the user calendar.', 'error');
-                }
+        // Initialise the calendar
+        $(document).trigger('gh.calendar.init', {'triposData': triposData});
 
-                var calendarData = {
-                    'triposData': triposData,
-                    'events': data
-                };
+        // Fetch the user's events
+        if (!gh.data.me.anon) {
 
-                $(document).trigger('gh.calendar.init', calendarData);
-            });
+            // Put the calendar on today's view
+            $(document).trigger('gh.calendar.navigateToToday');
```

Comment:
I thought this was the default behaviour, I might be wrong though. Also, why do we need to limit this to logged in users?